### ISSUE TITLE
Add the concept of a count summary widget

### ISSUE THREAD

**Issue by jeffbaranyi** [Esri/storymap-journal#26]
I would like to have a Map Journal with a feature count in the side panel that is driven from a feature service.  This is how the Summary Widget in the Operations Dashboard works - http://doc.arcgis.com/en/operations-dashboard/windows-desktop/author/summary-widget.htm.

The use case is list the current number of wildfires in a briefing.  I don't want to have to manually update that info for each briefing.  I'd like for that to be driven from the data.

See the attached screen shot for the concept.
![mapjournalcount](https://cloud.githubusercontent.com/assets/5030938/6474485/52ca85b0-c1be-11e4-9090-fb0101b2b223.png)

**Comment by glazou** [comment]
Interesting idea, thanks for the request. 

Unfortunately that kind of UI is a lot of work. I don't think we are going to be able to implement soon. 

Not what you need as this would made the statistics available through the Main Stage and not the Side Panel but I'm curious to know if you have you tried using the Impact Summary template and embedding it into Map Journal through a WebPage embed? The Web Operation Dashboard can also be embedded since last release http://blogs.esri.com/esri/arcgis/2015/03/03/whats-new-arcgis-online-march-2015/

**Comment by driskull** [comment]
Is there a way to include an iframe in the side panel? If so we could have a simple app to report the stats from a webmap/layer.

**Comment by glazou** [comment]
Yes you can do that through that button (or toggle the source view and enter your iframe)
![image](https://cloud.githubusercontent.com/assets/994078/6537350/cbc6c3c2-c40a-11e4-95f8-399a5371116e.png)

That could be a good first step but would require configuration and copy/pasting from the user. Do you see that as a possibility for a template in Online?

But at the end I think we want a fully integrated user experience in the app.

**Comment by driskull** [comment]
@jeffbaranyi should be doable then.
